packages/cli/package.json

```diff
@@ -1,20 +1,25 @@
 {
-  "name": "hops-cli",
+  "name": "hops",
   "version": "9.1.1",
   "description": "Global CLI module to initialize new hops projects",
   "keywords": ["hops", "cli"],
   "license": "MIT",
   "bin": {
-    "hops": "hops.js"
+    "hops": "hops.js",
+    "hops-start": "commands/start.js"
```

Comment:
I think I would prefer not exposing hops-start in the global scope. What do you think?